ci/requirements/environment.yaml

```
@@ -0,0 +1,20 @@
+name: xsar-tests
+channels:
+  - conda-forge
+dependencies:
+  - ipython
+  - pre-commit
+  - pytest
+  - pytest-reportlog
+  - pytest-cov
+  - numpy
+  - toolz
+  - cytoolz
+  - python-dateutil
+  - construct
+  - fsspec
+  - xarray
+  - dask
+  - distributed
+  - aiohttp
+  - requests
```

this should all dependencies to work properly, so:
```suggestion
  - requests
  - gdal
  - affine
  - rasterio
  - fiona
  - cartopy
  - pyproj
  - scipy
  - shapely
  - geopandas
  - pytz
  - psutil
  - jinja2
  - rioxarray
  - lxml
```

---

fixed.